dev-haskell/dbus/dbus-1.3.11.ebuild

```
@@ -15,27 +15,29 @@ LICENSE="Apache-2.0"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
 
+CABAL_CHDEPS=('template-haskell >= 2.18 && < 2.23' 'template-haskell >= 2.18 && < 2.24')
+
 RDEPEND="<dev-haskell/cereal-0.6:=[profile?]
 	>=dev-haskell/conduit-1.3.0:=[profile?] <dev-haskell/conduit-1.4:=[profile?]
-	<dev-haskell/lens-5.3:=[profile?]
-	>=dev-haskell/network-3.1.2.1:=[profile?] <dev-haskell/network-3.2:=[profile?]
+	<dev-haskell/lens-5.4:=[profile?]
+	>=dev-haskell/network-3.2:=[profile?] <dev-haskell/network-3.3:=[profile?]
```

Lots of packages still depend on `<network-3.2`. Is this addressed in this PR?

---

It might be worth it to try compiling with the older `network`.

```bash
CABAL_CHDEPS=(
	'network >= 3.2 && < 3.3' 'network >=3.1.2.1 && <3.2'
)
```

```suggestion
	>=dev-haskell/network-3.1.2.1:=[profile?] <dev-haskell/network-3.2:=[profile?]
```

---

Marking conversation as resolved as this PR has morphed into dealing with network-3.2.